sci-libs/hdf5/hdf5-1.12.1.ebuild

```diff
@@ -17,7 +17,7 @@ SRC_URI="https://www.hdfgroup.org/ftp/HDF5/releases/${MAJOR_P}/${MY_P}/src/${MY_
 LICENSE="NCSA-HDF"
 SLOT="0/${PV%%_p*}"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux"
-IUSE="cxx debug doc examples fortran +hl mpi szip test threads unsupported zlib"
+IUSE="cxx debug doc examples fortran +hl mpi ros3 szip test threads unsupported zlib"
```

Comment:
You're adding a new USE flag and changing RDEPEND, so we need to revbump (https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html)

Reply:
done :)